# fix for icons on dark theme and move tabs above urlbar
--- a/./configuration/user.js
+++ b/./configuration/user2.js
@@ -3,17 +3,19 @@
  */

 // Enable customChrome.css
-user_pref("toolkit.legacyUserProfileCustomizations.stylesheets", true);
+pref("toolkit.legacyUserProfileCustomizations.stylesheets", true);

 // Set UI density to normal
-user_pref("browser.uidensity", 0);
+pref("browser.uidensity", 0);

 // Enable SVG context-propertes
-user_pref("svg.context-properties.content.enabled", true);
+pref("svg.context-properties.content.enabled", true);

 // Disable private window dark theme
-user_pref("browser.theme.dark-private-windows", false);
+pref("browser.theme.dark-private-windows", false);

 // Enable rounded bottom window corners
-user_pref("widget.gtk.rounded-bottom-corners.enabled", true);
+pref("widget.gtk.rounded-bottom-corners.enabled", true);

+// Move tabs above urlbar
+pref("gnomeTheme.tabsAsHeaderbar", true);
